src/app/components/Account/index.tsx

```diff
@@ -160,3 +169,84 @@ export const Account: FC<AccountProps> = ({ account, token, isLoading, tokenPric
     </>
   )
 }
+
+export type ConsensusAccountDataProps = {
+  account?: Account
+  isLoading?: boolean
+  showLayer?: boolean
+  standalone?: boolean
+}
+
+export const ConsensusAccountData: FC<ConsensusAccountDataProps> = ({
```

Comment:
What are components with "*Data" suffix?

Reply:
Unfortunately, we don't have any consistent convention for naming all our components (that I am aware of).

In case of these two components, they simply display the data in the fields of the object. `RuntimeAccountData` displays a `RuntimeAccount`. Earlier this React component used to be called `Account`, but that's not really a good name since we also have an Account type in the data layer (for Consensus accounts.) `ConsensusAccountData` used to be implemented as part of the page code for displaying consensus account details; it has been factored out in #1422.

Reply:
ah idk :shrug:
- we use "*View" for mostly stateless subsets of components and "*Page" for non-reusable
- "*Data" for component reminds me of renderless components (before hooks existed)
- out of RuntimeAccount and RuntimeAccountData I would expect RuntimeAccountData to be the data layer type

Reply:
Yeah, it is a true that "*Data" invokes the expectation of finding logic and data storage. The reason I didn't go with "*View" in this case is that we already have some components called `RuntimeAccountView`, so I wanted something else. On the other hand, we can probably do with one less layer of wrappers (between the page and the actual data display), and in that case, the only remaining class could still be called "*View". Let me try that in a separate follow-up PR.

Reply:
See fix in https://github.com/oasisprotocol/explorer/pull/1423.